docs/reference/ml/anomaly-detection/apis/get-job.asciidoc

```diff
@@ -81,6 +81,69 @@ monitor progress.
 (string) The time the job was created. For example, `1491007356077`. This
 property is informational; you cannot change its value.
 
+`datafeed`::
```

Comment:
I am thinking it might make UI integration and consistency of what users see during upgrades better if this is called `datafeed_config`.

At the moment the UI already pretends that the datafeed config is nested inside the job config in the JSON panel of the jobs list, but nests it in an object named `datafeed_config`, not `datafeed`.

So for what the user sees to remain consistent it would be better if the nested object added to the job config by the backend code was called `datafeed_config`.  Otherwise in the period when the backend might be returning `datafeed` and the UI is still getting the datafeed separately the JSON panel in the jobs list in the UI will show both `datafeed` _and_ `datafeed_config`.

The one potential problem here would be if the UI currently throws an error if a job config JSON already contains `datafeed_config` instead of simply overwriting it with a separately obtained `datafeed_config`.  If the UI throws an error in this situation then we'll have to stick with `datafeed` and tolerate the duplication in versions of the UI that are already released.  If the UI throws an error when `datafeed_config` is already present we cannot fix this now, as the upgrade rules say that Elasticsearch must be upgraded before Kibana, and so already released UI code from 7.0 to 7.13 must work with 7.14 - 7.last Elasticsearch.

/cc @jgowdyelastic @peteharverson

Reply:
I am fine changing it to `datafeed_config`. 

> The one potential problem here would be if the UI currently throws an error if a job config JSON already contains `datafeed_config`

That would be a problem indeed. Eagerly awaiting feed back @jgowdyelastic @peteharverson :)